### PR TITLE
Fixed inverted zooming.

### LINKED ISSUE #3568: Zoom button has inverted functionality
This issue is about Piano Roll, Automation Editor and Song Editor. Before it worked same way as in wiki: https://lmms.io/documentation/Song_Editor#Zoom_Control

Scenario 1: For example in piano roll current zoom is 100%. To see notes a little bit closer I want to zoom in. I use mouse left button to click the "100%" button. I expect piano roll to zoom in because in most programs (Audacity etc) left mouse click zooms in. LMMS zooms out instead.

Scenario 2: For example in piano roll current zoom is 100%. To see all notes I want to zoom out. I use mouse right button to click the "100%" button. I expect piano roll to zoom out because in most programs right mouse click zooms out. LMMS zooms in instead.

This bothers me a lot. Before I make a pull request I want to be sure it's a bug not a new feature.

Ctrl + mouse scroll has expected behaviour (fixed in #3273).

### PR DESCRIPTION
**Pull request by karmux**
Fixes #3568.

### CODE REVIEW

File: src/gui/editors/AutomationEditor.cpp [hunk 1]
```diff
@@ -1627,11 +1627,11 @@ void AutomationEditor::wheelEvent(QWheelEvent * we )
 		int x = m_zoomingXModel.value();
 		if( we->delta() > 0 )
 		{
-			x--;
+			x++;
 		}
 		if( we->delta() < 0 )
```
Comment (zapashcanon) [inline src/gui/editors/AutomationEditor.cpp]:
```c++
else if( we->delta() < 0 )
```

Will prevent one useless test when `x > 0`. I know it's not directly related to this PR, but spotted this so...

File: src/gui/editors/PianoRoll.cpp [hunk 2]
```diff
@@ -3281,11 +3281,11 @@ void PianoRoll::wheelEvent(QWheelEvent * we )
 		int z = m_zoomingModel.value();
 		if( we->delta() > 0 )
 		{
-			z--;
+			z++;
 		}
 		if( we->delta() < 0 )
 		{
-			z++;
+			z--;
```
Comment (zapashcanon) [inline src/gui/editors/PianoRoll.cpp]: Same here...

File: src/gui/editors/SongEditor.cpp [hunk 3]
```diff
@@ -361,11 +361,11 @@ void SongEditor::wheelEvent( QWheelEvent * we )
 
 		if( we->delta() > 0 )
 		{
-			z--;
+			z++;
 		}
 		if( we->delta() < 0 )
 		{
-			z++;
+			z--;
```
Comment (zapashcanon) [inline src/gui/editors/SongEditor.cpp]: And here too.

**Comment by qnebra** [comment]
Tested locally, lmms not crashing yet. LMB zoom in, RMB zoom out both on pianoroll and automation editor, and song editor too.

**Comment by zonkmachine** [comment]
Merge?

**Comment by tresf** [comment]
Cherry-picked and ready for master via 2652005.

**Pull request merged**